### PR TITLE
Update deployment_guide.md

### PR DESCRIPTION
**Pull request by SrinivasChilveri**
The environment will not bet up without creating the pv and without nfs required software on k8s nodes

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

File: kahu/deployment_guide.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ tags: ["deployment guide", "kahu", "soda-cdm"]
 Kahu features can be realised through deployment on kubernetes(k8s) cluster. 
 If you do not have a k8s cluster, please refer [kubernetes setup documentation](https://kubernetes.io/docs/setup) to setup one.
 
+Note: Need to install the required things for nfs server on the k8s nodes (Example: sudo apt install nfs-kernel-server on ubuntu)
```
Comment (joseph-v) [inline kahu/deployment_guide.md:13]: Is this necessary as we use NFS Server from docker image with all dependencies installed?

File: kahu/deployment_guide.md [hunk 2]
```diff
@@ -40,21 +42,26 @@ kubectl get service -n kahu | grep nfs-server
 ```
 6) Edit nfs-pv.yaml to update NFS-SERVER-IP-ADDR with service ip address obtained in step 5
 
-7) Create nfs server deployment
+7) create pv
```
Comment (sushanthakumar) [inline kahu/deployment_guide.md]: Suggest to change to "Create"

**Review by sushanthakumar — APPROVED** [review]
LGTM

**Review by joseph-v — APPROVED** [review]
LGTM

**Pull request merged**